Create Main page (#81)
nebari-dev/nebari: docs/docs/welcome.mdx

- [Jupyter Videochat](https://github.com/yuvipanda/jupyter-videochat): allows video-chat with JupyterHub peers inside JupyterLab, powered by Jitsi.
- [Conda Store](https://github.com/quansight/conda-store): serves identical conda environments and controls its life-cycle.
- [Conda Docker](https://github.com/conda-incubator/conda-docker): an extension to the docker concept of having declarative environments that are associated with Docker images
  allowing tricks and behavior that otherwise would not be allowed.
- [VS Code codeserver](https://github.com/cdr/code-server): built-in web editor tied to jupyterlab server

### Integrations

Nebari does all the heavy lifting of integrating with common services. 

- [**Prefect**](https://www.prefect.io/) - workflow management
- [**ClearML**](https://clear.ml/) - machine learning experiment tracking and analysis
- [**Prometheus**](https://prometheus.io/) - cluster monitoring
- [**Grafana**](https://grafana.com/) - cluster monitoring visualizations
- [**CDS Dashboards**](https://cdsdashboards.readthedocs.io/) - shared dashboarding across users

## Why use Nebari?

> Nebari provides enables teams to build their own scalable compute infrastructure with:

Reviewer: ```suggestion
Nebari enables teams to build their own scalable compute infrastructure with:
```

> Straightforward installation and maintenance controlled by a single configuration file.

Reviewer: ```suggestion
- Straightforward installation and maintenance controlled by a single configuration file.
```

> Straightforward installation and maintenance controlled by a single configuration file.

Reviewer: ```suggestion
straightforward installation and maintenance controlled by a single configuration file.
```

---
slug: /
title: Welcome to Nebari's documentation
---

export const invisible_table = {
   color: 'blue',
   lineHeight: 10,
   padding: '1.5em',
   background: 'white',
   backgroundColor: 'white',
}

:::warning We are currently undergoing a rename from [QHub](https://docs.qhub.dev/) to Nebari.   
:construction: Please excuse our dust as we build out our new docs :) :construction:
:::

> [INSERT NEBARI LOGO HERE]

Reviewer: Since we will not have the logo before the sprint let's remove this as this looks unfinished

> Nebari: The Roots of a Bonsai Tree

Reviewer: Is this needed? unless it adds value/context (which seems like it does not as it is written)

## Nebari Vision

An opinionated open source [JupyterHub][jupyterhub] distribution that provides 
an all-in-one development and experimentation platform for teams to work 
efficiently and collaboratively.

> Deployment, integration and maintenance is minimized by using the 

Reviewer: ```sugestion
Deployment, integration, and maintenance is minimized by using the 
```

> infrastructure as code (IaC) management approach.

Reviewer: ```sugestion
Infrastructure as Code (IaC) management approach.
```

## Nebari has a *DevOps for non-DevOps people* approach.

> ![Nebari overview sequence](/img/nebari_overview_sequence.png)

Reviewer: TODO: needs a better alt -text description

> ![Nebari overview sequence](/img/nebari_overview_sequence.png)

Author: ```suggestion
![Nebari overview sequence showing deploying from yaml to cloud resources to the instance spinup selection](/img/nebari_overview_sequence.png)
```

### Components

The technology stack is an integration of the following existing open source libraries:

> - [**Terraform**](https://www.terraform.io/intro/index.html): a tool for building, changing, and versioning infrastructure.

Reviewer: Is the **bold** text necessary here? all these will be links so already distinctive from the surrounding text

- [**Kubernetes**](https://kubernetes.io/docs/home/): a cloud-agnostic orchestration system
- [**Helm**](https://helm.sh/): a package manager for Kubernetes
- [**JupyterHub**](https://jupyter.org/hub): a shareable compute platform for data science
- [**JupyterLab**](https://jupyterlab.readthedocs.io/en/stable/): a web-based interactive development environment for Jupyter Notebooks
- [**Dask**](https://docs.dask.org/en/latest/): a scalable and flexible library for parallel computing in Python
  - [**Dask-Gateway**](https://gateway.dask.org/): a secure, multi-tenant server for managing Dask clusters

> - [**Keycloak**](https://www.keycloak.org/): Open Source Identity and Access Management

Reviewer: this one follows a Title Case while the others seem to be natural sentences/

- [**GitHub Actions**](https://docs.github.com/en/actions): a tool to automate, customize, and execute software development workflows in a GitHub repository.
- [**traefik**](https://traefik.io/): for routing web/tcp traffic inside cluster
  - [**traefik-forward-auth**](https://github.com/thomseddon/traefik-forward-auth): for single sign on and easy securing of web applications

Amongst the newly created open source libraries on the tech stack are:

> - [**Jupyterhub SSH**](https://github.com/yuvipanda/jupyterhub-ssh): brings the SSH experience to a modern cluster manager.

Reviewer: Same comment as above the **bold** seems unneccessary here

> - [**Jupyter Videochat**](https://github.com/yuvipanda/jupyter-videochat): allows video-chat with JupyterHub peers inside JupyterLab, powered by Jitsi.

Reviewer: @dharhas - we discussed getting this out of nebari *soon* shall we remove the mention?

> - [**Terraform**](https://www.terraform.io/intro/index.html): a tool for building, changing, and versioning infrastructure.
> - [**Kubernetes**](https://kubernetes.io/docs/home/): a cloud-agnostic orchestration system
> - [**Helm**](https://helm.sh/): a package manager for Kubernetes
> - [**JupyterHub**](https://jupyter.org/hub): a shareable compute platform for data science
> - [**JupyterLab**](https://jupyterlab.readthedocs.io/en/stable/): a web-based interactive development environment for Jupyter Notebooks
> - [**Dask**](https://docs.dask.org/en/latest/): a scalable and flexible library for parallel computing in Python
>   - [**Dask-Gateway**](https://gateway.dask.org/): a secure, multi-tenant server for managing Dask clusters
> - [**Keycloak**](https://www.keycloak.org/): Open Source Identity and Access Management
> - [**GitHub Actions**](https://docs.github.com/en/actions): a tool to automate, customize, and execute software development workflows in a GitHub repository.
> - [**traefik**](https://traefik.io/): for routing web/tcp traffic inside cluster
>   - [**traefik-forward-auth**](https://github.com/thomseddon/traefik-forward-auth): for single sign on and easy securing of web applications
> 
> Amongst the newly created open source libraries on the tech stack are:
> 
> - [**Jupyterhub SSH**](https://github.com/yuvipanda/jupyterhub-ssh): brings the SSH experience to a modern cluster manager.
> - [**Jupyter Videochat**](https://github.com/yuvipanda/jupyter-videochat): allows video-chat with JupyterHub peers inside JupyterLab, powered by Jitsi.
> - [**Conda Store**](https://github.com/quansight/conda-store): serves identical conda environments and controls its life-cycle.
> - [**Conda Docker**](https://github.com/conda-incubator/conda-docker): an extension to the docker concept of having declarative environments that are associated with Docker images
>   allowing tricks and behaviour that otherwise would not be allowed.
> - [**VS Code codeserver**](https://github.com/cdr/code-server): built-in web editor tied to jupyterlab server

Author: ```suggestion
- [Terraform](https://www.terraform.io/intro/index.html): a tool for building, changing, and versioning infrastructure.
- [Kubernetes](https://kubernetes.io/docs/home/): a cloud-agnostic orchestration system
- [Helm](https://helm.sh/): a package manager for Kubernetes
- [JupyterHub](https://jupyter.org/hub): a shareable compute platform for data science
- [JupyterLab](https://jupyterlab.readthedocs.io/en/stable/): a web-based interactive development environment for Jupyter Notebooks
- [Dask](https://docs.dask.org/en/latest/): a scalable and flexible library for parallel computing in Python
  - [Dask-Gateway](https://gateway.dask.org/): a secure, multi-tenant server for managing Dask clusters
- [Keycloak](https://www.keycloak.org/): an open source identity and access management tool
- [GitHub Actions](https://docs.github.com/en/actions): a tool to automate, customize, and execute software development workflows in a GitHub repository.
- [traefik](https://traefik.io/): for routing web/tcp traffic inside cluster
  - [traefik-forward-auth](https://github.com/thomseddon/traefik-forward-auth): for single sign on and easy securing of web applications

Amongst the newly created open source libraries on the tech stack are:

- [Jupyterhub SSH](https://github.com/yuvipanda/jupyterhub-ssh): brings the SSH experience to a modern cluster manager.
- [Jupyter Videochat](https://github.com/yuvipanda/jupyter-videochat): allows video-chat with JupyterHub peers inside JupyterLab, powered by Jitsi.
- [Conda Store](https://github.com/quansight/conda-store): serves identical conda environments and controls its life-cycle.
- [Conda Docker](https://github.com/conda-incubator/conda-docker): an extension to the docker concept of having declarative environments that are associated with Docker images
  allowing tricks and behavior that otherwise would not be allowed.
- [VS Code codeserver](https://github.com/cdr/code-server): built-in web editor tied to jupyterlab server
```

### Integrations

Nebari does all the heavy lifting of integrating with common services. 

- [**Prefect**](https://www.prefect.io/) - workflow management
- [**ClearML**](https://clear.ml/) - machine learning experiment tracking and analysis
- [**Prometheus**](https://prometheus.io/) - cluster monitoring
- [**Grafana**](https://grafana.com/) - cluster monitoring visualizations
- [**CDS Dashboards**](https://cdsdashboards.readthedocs.io/) - shared dashboarding across users

## Why use Nebari?

Nebari provides enables teams to build their own scalable compute infrastructure with:

> - Easy installation and maintenance controlled by a single configuration file.

Reviewer: Avoid terms such as easy, just, simply... 

```sugestion
Straightforward installation and maintenance controlled by a single configuration file.
```

> - Autoscaling JupyterHub installation deployed on the Cloud provider of your choice.

Reviewer: Maybe need to add a disclaimer or note of the cloud providers supported in case someone's choice is something like OVH

> - Autoscaling JupyterHub installation deployed on the Cloud provider of your choice.

Author: ```suggestion
- Autoscaling JupyterHub installation deployed on one of the supported Cloud providers (AWS, GCP, DO, or Azure).
```

> - Option to choose from multiple compute instances, such as: **normal**, **high memory**, **GPU**, etc.

Reviewer: As a new user this makes no sense to me - what is a big, small, high mem instance? suggest removing from, such as onwards

> - Option to choose from multiple compute instances, such as: **normal**, **high memory**, **GPU**, etc.

Author: ```suggestion
- Option to choose from multiple compute instances.
```

- Autoscaling Dask compute clusters for big data using any instance type.
- Shell access and remote editing access (VSCode remote) through KubeSSH.
- Shared filesystem allowing users to work on projects privately, within groups, or across the organisation.
- Robust compute environment handling allowing both prebuilt and ad-hoc environment creation.

> - Integrated video conferencing, using [Jitsi](https://meet.jit.si/).

Reviewer: Same comment as above

> <!-- 

Reviewer: If this is not applicable please remove

> <!-- 
> ![handshake icon](/img/icon_handshake.jpg) Quickly Deploys a Jupyterhub environment where data scientists can collaborate  
> ![gear icon](/img/icon_gear.jpg) Provides a foundation for deploying additional technologies  
> ![chart icon](/img/icon_chart.jpg) Enables scalability beyond the limits of local machines  
> ![cloud icon](/img/icon_cloud.jpg) Cloud deployment on multiple cloud platforms with minimal experience   
> -->

Author: ```suggestion
```
